### PR TITLE
5pm 4 ge download as csv

### PR DESCRIPTION
**Pull request by codysasaki**

### CODE REVIEW

**Review by at527 — APPROVED** [review]
LGTM

File: javascript/src/main/pages/History/Ge.js [hunk 1]
```diff
@@ -24,13 +26,18 @@ const Ge = () => {
     //const [courseJSON, setCourseJSON] = useState('{"course" : "cs148"}');
     // const courseJSON = '{"course" : "cs156"}';
     const [courseJSON, setCourseJSON] = useState(initialCourseJSON);
-
+    const courseHeaders = [
+        { label: "courseId", key: "courseId" },
+        { label: "title", key: "title" },
+        { label: "units", key: "unitsFixed" }
+    ]
```
Comment (at527) [inline javascript/src/main/pages/History/Ge.js:33]: Should it download all the displayed columns of the search? The deployment doesn't seem to do that.

**Review by at527 — CHANGES_REQUESTED** [review]

**Comment by zminshall** [comment]
Please add a description to this PR and link it to an issue. I am also interested in seeing more of the data added to the downloaded csv. Documentation for adding more columns available at https://www.npmjs.com/package/react-csv

**Review by zminshall — CHANGES_REQUESTED** [review]
This PR still needs a description and needs to be linked to an issue.

**Pull request closed without merging**